### DISCUSSION TITLE
Citing aesara

### DISCUSSION THREAD

**Discussion by richardnaab** [aesara-devs/aesara#1504] [Q&A] [answered]
Hi all,
We're using aesara for statistical modeling in neutrino astronomy (IceCube measurements of the astrophysical diffuse neutrino flux) and would like to give a reference to the library. Is there some preferred way of citation?
Thanks!

**Comment by brandonwillard** [comment]
It's always great to hear about how this work is used by others, thanks!

I don't think we have a proper citation reference, but we can try to come up with one shortly.

**Reply by richardnaab** [reply]
Thanks,
Did you, by chance, come up if one already?
For now, I have put a citation that will appear as 
> [18] Aesara-Developers, “aesara,” 2023. https://github.com/aesara-devs/aesara.

but also have a couple of days left before the conference organizers want the proceedings to be uploaded. Corrections to the proceedings can also be made afterwards.

**Reply by brandonwillard** [reply, marked as answer]
Sorry for the slow response!  Here's a BibTeX entry:
```
@software{WillardAesara2023,
  title = {Aesara},
  shorttitle = {Aesara},
  author = {Willard, Brandon T. and Louf, R{\'e}mi and Chaudhari, Kaustubh and Mares, Ben and Lunagariya, Smit and {Foreman-Mackey}, Dan and Gerlanc, Daniel},
  year = {2023},
  address = {{GitHub}},
  url = {https://github.com/aesara-devs/aesara},
  urldate = {2023-07-11},
  abstract = {Aesara is a Python library for defining, optimizing, and efficiently evaluating mathematical expressions involving multi-dimensional arrays.},
  howpublished = {Aesara Developers},
  keywords = {aesara,automatic-differentiation,symbolic-computation,tensors,theano,transpiler}
}
```
The authors list was derived based on current group/org membership and [contributions](https://github.com/aesara-devs/aesara/graphs/contributors?from=2020-01-08&to=2023-07-11&type=c).  We have a paper in the works, and we'll likely use that as the reference, but, for now, this should work.

**Reply by brandonwillard** [reply]
@aesara-devs, if I missed anyone, someone doesn't want to be listed, someone isn't showing up on that [contributions](https://github.com/aesara-devs/aesara/graphs/contributors?from=2020-01-08&to=2023-07-11&type=c) page, or anyone has other recommendations, please tell me so I can fix it.

**Reply by brandonwillard** [reply]
Oh, I forgot, there's also this: https://zenodo.org/record/7662296.  We should update it, though.
